[2024-08-10T16:17:33.187+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-10T16:17:33.205+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T13:17:28.100388+00:00 [queued]>
[2024-08-10T16:17:33.216+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T13:17:28.100388+00:00 [queued]>
[2024-08-10T16:17:33.217+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-10T16:17:33.244+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_air_pollution> on 2024-08-10 13:17:28.100388+00:00
[2024-08-10T16:17:33.251+0300] {standard_task_runner.py:64} INFO - Started process 865 to run task
[2024-08-10T16:17:33.256+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'pollution_air_dags', 'extract_air_pollution', 'manual__2024-08-10T13:17:28.100388+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_dags.py', '--cfg-path', '/tmp/tmpi3qe45yq']
[2024-08-10T16:17:33.258+0300] {standard_task_runner.py:91} INFO - Job 111: Subtask extract_air_pollution
[2024-08-10T16:17:33.329+0300] {task_command.py:426} INFO - Running <TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T13:17:28.100388+00:00 [running]> on host DESKTOP-DKUGVOH.
[2024-08-10T16:17:33.444+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pollution_air_dags' AIRFLOW_CTX_TASK_ID='extract_air_pollution' AIRFLOW_CTX_EXECUTION_DATE='2024-08-10T13:17:28.100388+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-10T13:17:28.100388+00:00'
[2024-08-10T16:17:33.447+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-10T16:17:33.449+0300] {logging_mixin.py:188} INFO - Le fichier 'air_pollution_data.csv' est introuvable. Assurez-vous qu'il est bien dans le mÃªme dossier que ce script.
[2024-08-10T16:17:33.470+0300] {python.py:237} INFO - Done. Returned value was: None
[2024-08-10T16:17:33.471+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-10T16:17:33.477+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pollution_air_dags, task_id=extract_air_pollution, run_id=manual__2024-08-10T13:17:28.100388+00:00, execution_date=20240810T131728, start_date=20240810T131733, end_date=20240810T131733
[2024-08-10T16:17:33.511+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-10T16:17:33.539+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-10T16:17:33.552+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
